frappe/patches/v11_0/change_email_signature_fieldtype.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2018, Frappe Technologies Pvt. Ltd. and Contributors
+# MIT License. See license.txt
+
+import frappe
+
+def execute():
+	signatures = frappe.db.get_list('User', {'email_signature': ['!=', '']},['name', 'email_signature'])
+	frappe.reload_doc('core', 'doctype', 'user')
+	for d in signatures:
+		signature = d.get('email_signature')
+		signature = signature.replace('\n', '<br>')
```

Comment:
`signature = frappe.markdown(signature)` ?

Reply:
`frappe.markdown` is written on client side, so can't use it in patch.

Reply:
also exists in server side!